js/components/addEditBookmark.js

```diff
@@ -115,8 +115,8 @@ class AddEditBookmark extends ImmutableComponent {
             </select>
           </div>
           <div className='formRow'>
-            <span />
             <Button l10nId='save' className='primaryButton' onClick={this.onSave} />
+            <Button l10nId='cancel' className='subtleButton cancelButton' onClick={this.onClose} />
```

Comment:
I'm not sure if this is the optimal order from a viewpoint of UX. "Save" "Cancel" or "Cancel" "Save"?